query-service-impl/src/main/java/org/hypertrace/core/query/service/pinot/converters/PinotFunctionConverterConfig.java

```diff
@@ -58,4 +68,9 @@ public String getDistinctCountFunction(String arg) {
     return Optional.ofNullable(distinctCountAggOverrides.get(arg))
         .orElse(distinctCountAggregationFunction);
   }
+
+  public String getDistinctCountMvFunction(String arg) {
+    return Optional.ofNullable(distinctCountAggOverrides.get(arg))
```

Comment:
Given that we have introduced separate configs for Agg function (SV vs MV), we should extend the same to overrides as well. Instead of reusing `distinctCountAggOverrides` for both Sv and Mv, we can have 2 separate overrides config: `distinctCountAggOverrides` and `distinctCountMvAggOverrides`. Will be error-free and easier to understand.

Reply:
The idea here is to have overrides at the handler level based on the data type, whether it's a single value field or a multi-value field. The `distinctCountAggOverrides` allows for overriding specific attributes, which can be of any data type as it maps an attribute to a function.

Reply:
Single override will also work. Just that the onus of configuring correct flavour of function (SV vs MV) is on user. 
```
distinctCountAggOverrides = {
 "attr1" = DISTINCTCOUNT
"attr2" = DISTINCTCOUNT
}
```
If suppose attr2 is Mv, then the only valid alternative is DISTINCTCOUNTMV. Backend can only validate if there is some attribute metadata (SV vs MV) available (not sure about it).